deployer/scripts/uuid_migrate.sh

```diff
@@ -88,7 +91,7 @@ function run_uuid_migration() {
     [[ "${OPS_PROJECTS[@]}" =~ "$index" ]] && continue
 
     uid=$(oc get project "$index" -o jsonpath='{.metadata.uid}')
-    create_alias $index $uid
+    create_alias ${PROJ_PREFIX}$index $uid
```

Comment:
Yeah, it looks like it didn't find that record...  `++ [[ -z '' ]]`
I'd be curious what the logs looked like in the deployer and what log indices were in that ES instance..

It could be that the uuid migration didn't run successfully...